docs/tutorials/creating-javascript-app-using-alfresco-js-api.md

```diff
@@ -0,0 +1,100 @@
+---
+Level: Basic
+---
+# Creating your Alfresco JavaScript application
+
+In this tutorial you are going to see how to create from scratch an application in JavaScript to interact with Alfresco. This is a "getting started" task that should enable you to start developing your own JavaScript application on top of Alfresco Content Services or Alfresco Process Services.
+
+In this tutorial you are going to learn how to develop on top of Alfresco Content Services, but the development on top of Alfresco Process Services follows exactly the same principle.
+
+**Note:** The development on top of Alfresco Content Services AND Alfresco Process Services has the only limitation of the [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing). In this case you might need to proxy your application first.
+
+## Prerequites
+
+The only prerequisite of this tutorial is the availability of an instance of Alfresco Content Services in a [Docker](https://www.docker.com/) container. Docker is not the only option you might have for the deployment, but it is the faster we found to focus on the development instead of the environment setup.
+
+If you don't have an instance of Alfresco Content Services up and running check the [preparation of the development environment](./preparing-the-development-environment.html).
+
+Only to download the requested libraries, you will need the `npm` client. Also in this case check the [preparation of the development environment](./preparing-the-development-environment.html) for further details on how to install it into your development environment.
+
+## Creating the JavaScript application
+
+Assuming that you have your Alfresco Content Services instance up an running at `http://localhost:8082/alfresco`, let' see here how to develop a JavaScript application from scratch. The JavaScript application will be able to interact with Alfresco back-end services using the [`alfresco-js-api`](https://github.com/Alfresco/alfresco-js-api) library. This library does not necessarily require to be used into an Angular application but it is "framework agnostic".
```

Comment:
"up *and* running..." and "let'*s* see here.."

Reply:
Changed!